### PR TITLE
Footer

### LINKED ISSUE #3: Blog: footer
Goal: Implement the blog page footer
Note: there is one global footer for all hub sites and pages. there is no footer for blogs only.

Ref: https://news.sap.com/2023/12/being-human-in-the-age-of-ai/ > black footer

### PR DESCRIPTION
**Pull request by sdmcraft**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #3 

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/blog/2023/11/supply-chain-issues-keeping-ceos-up-at-night
- After: https://footer--hlx-test--urfuwo.hlx.live/blog/2023/11/supply-chain-issues-keeping-ceos-up-at-night

Pending Items:
- Url links
- Reuse global tokens
- Social links

### CODE REVIEW

File: blocks/footer/footer.css
Comment (pahupe) [inline blocks/footer/footer.css:1]: I believe for the language selector:
* the background color should be set to --udexColorGrey2 (same as backgrond color)
* the border radius of the selection box should be set to 4px (looking at https://www.figma.com/file/oSetT4LbatRmXlcB2A7V8Z/Content-Hubs-2024?type=design&node-id=269-30889&mode=design&t=U7BAtOyvjXZhn68k-0)

Comment (pahupe) [inline blocks/footer/footer.css:1]: In the SAP / contact element, the first line of the contact element (the US phone number) should use a different font, this one:
  --udexTypographyHeadingMediumXXSFontSize: 20px;
  --udexTypographyHeadingMediumXXSFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
  --udexTypographyHeadingMediumXXSFontWeight: 500;
  --udexTypographyHeadingMediumXXSLineHeight: 22px;

File: blocks/link-group/link-group.css
Comment (pahupe) [inline blocks/link-group/link-group.css:1]: The link group headline should have a padding-top of 0 to align it at the top with the logo and the language selection box (see https://www.figma.com/file/oSetT4LbatRmXlcB2A7V8Z/Content-Hubs-2024?type=design&node-id=269-30889&mode=design&t=U7BAtOyvjXZhn68k-0)

.block.link-group > div:first-of-type > div 
^ should not have a padding-top

For the same reason, the headline element itself (the <strong> tag) should not have a padding-top either.

File: blocks/link-group/link-group.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  line-height: 21px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.block.link-group.active ul {
+  display: block;
+}
+
+.block.link-group strong {
```
Comment (pahupe) [inline blocks/link-group/link-group.css:37]: The font definition should be using these design tokens, i.e. var(--...) :
  --udexTypographyNavigationMediumXLFontSize: 16px;
  --udexTypographyNavigationMediumXLFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
  --udexTypographyNavigationMediumXLFontWeight: 500;
  --udexTypographyNavigationMediumXLLineHeight: 18px;

File: blocks/link-group/link-group.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+.block.link-group ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: none;
+}
+
+.block.link-group ul li {
```
Comment (pahupe) [inline blocks/link-group/link-group.css:24]: Should be replaced with design tokens / variables:
  --udexTypographyBodyXSFontSize: 14px;
  --udexTypographyBodyXSFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
  --udexTypographyBodyXSFontWeight: 400;
  --udexTypographyBodyXSLineHeight: 21px;

File: blocks/footer/footer.css
Comment (sdmcraft) [reply blocks/footer/footer.css:1]: I am following the [footer design spec here](https://www.figma.com/file/OFEfvhyXJvYJ6MQZ30699m/v2.0-Brand-Digital-Design-System?type=design&node-id=8959-40002&mode=design&t=wWdPEMfKGZ0vivmW-0).

The background color per it is white.
I'd fix the border radius.

**Review by pahupe — APPROVED** [review]
Reviewed and approved.
Remaining issues will be added to a new footer ticker (iteration 2)

**Pull request merged**